research-hub-web/src/app/components/cards/standard-card/standard-card.component.html

```diff
@@ -25,13 +25,15 @@
         contentItem.slug
       ]"
     >
-      {{ contentItem.title }}
+      <span>{{ contentItem.title }}
+        &nbsp;
+        <mat-icon *ngIf="contentItem.ssoProtected" matTooltip="SSO Login Required">
```

Comment:
Suggest we remove SSO - as this is not really necessary and might be confusing to some people

Reply:
Might need to discuss with team? For users that don't have access I'd say it's needed so they can understand why they can't access this content. But we haven't collected specific user feedback about it so can't be sure.

Reply:
@cameronmclean what do you think?